src/incogniaApi.ts

```diff
@@ -100,13 +102,49 @@ export class IncogniaApi {
     })
   }
 
+  async registerWebSignup(
+    props: RegisterWebSignupProps
+  ): Promise<WebSignupResponse> {
+    const { sessionToken } = props || {}
+    if (!sessionToken) {
+      throw new IncogniaError('No sessionToken provided')
+    }
+
+    const data = convertObjectToSnakeCase(props)
+    return this.resourceRequest({
+      url: apiEndpoints.SIGNUPS,
+      method: Method.Post,
+      data
+    })
```

Comment:
What do you think about extracting a `registerSignup` method, similar to what is used on `transaction`?

Reply:
There is already a method called `registerSignup`, which is the mobile version.

However I could have a base method for Signup. Need to think a good name. Maybe `registerBaseSignup`?

Reply:
a9d7cd9..2773f90